setup/install/providers/azure.md

```diff
@@ -79,11 +89,19 @@ hal config provider azure account add my-azure-account \
   --subscription-id $SUBSCRIPTION_ID \
   --default-key-vault $VAULT_NAME \
   --default-resource-group $RESOURCE_GROUP \
+  --packer-resource-group $RESOURCE_GROUP \
   --app-key
 ```
 
-> NOTE: You will be prompted for the App Key on standard input. If necessary,
+> NOTE:
+> 1. You will be prompted for the App Key on standard input. If necessary,
 you can generate a new key: `az ad sp credential reset --name $APP_ID`
+> 2. Starting from Halyard v1.19, SSH public key will be used to provision VM scale set by default. If you prefer to use password, add the parameter\
+`--useSshPublicKey "false" \`\
+Prior to Halyard v1.19, the default credential option is password and halyard does not support the above parameter to switch the option to SSH public key.\
+Secret values of either the SSH public key or password are stored in the Azure key vault specified by the parameter `--default-key-vault`, in which the stored secret names are called "VMPassword" and "VMSshPublicKey", separately.
+> 3. The used Azure region by default is "eastus" and "westus". If you would like to add custom regions, add the parameter\
```

Comment:
Wording is a little funky. Should read "The Azure region used by default are " ...

Reply:
> Wording is a little funky. Should read "The Azure region used by default are " ...

resolved